hack/make/.ensure-syscall-test

```diff
@@ -4,5 +4,20 @@ set -e
 # Build a C binary for cloning a userns for seccomp tests
 # and compile it for target daemon
 if [ "$DOCKER_ENGINE_GOOS" = "linux" ]; then
-	docker build ${DOCKER_BUILD_ARGS} -qt syscall-test contrib/syscall-test > /dev/null
+	if [ "$DOCKER_ENGINE_GOOS" = "$DOCKER_CLIENT_OSARCH" ]; then
+		tmpdir=$(mktemp -d /tmp/XXXXXXXXXX)
```

Comment:
Isn't just `mktemp -d` enough here? (It should default to the appropriate `/tmp` given the invoker's preference specified in `TMPDIR`, right?)